src/scss/components/form/_input.scss

```diff
@@ -0,0 +1,47 @@
+@charset "UTF-8";
+// Input
+// - - - - - - - - - - - - - - - - - - - - - - - - -
+
+#{$all-text-inputs} {
+  @include pad(0 0 0 0);
+  @include marg(0 0 default 0);
+  width: 100%;
```

Comment:
Properties should be ordered border-radius, box-shadow, display, outline, transition, width